config/locales/jobseekers.yml

```
@@ -480,6 +498,9 @@ en:
         warning: You cannot review a non-draft job application
         training_and_cpds:
           none: You have not added any training.
+        religious_information:
+          heading: The school is a school with a religious character, please therefore provide the additional information requested as appropriate.
```

```suggestion
          heading: This school has a religious character and has asked applicants to provide religious information as part of the application, where possible.
```
could we tweak to the following please @starswan :)